x-pack/plugins/infra/public/components/log_stream/log_stream.stories.mdx

```diff
@@ -431,3 +460,7 @@ class MyPlugin {
   endTimestamp={...}
 />
 ```
+
+### Setting component height
+
+It's possible to pass a `height` prop, e.g. `60vh` or `300px`, to specify how much vertical space the component should consume.
```

Comment:
What happens when you don't set this value, i.e. what's the default value/unit? Anything? I forget how HTML works :P

Reply:
We use a default of 400px, which gets set here if needed: https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/public/components/log_stream/log_stream.tsx#L76

That looks fine as a "widget". It's likely most solutions will override this.